src/main/java/core/basesyntax/service/WriterService.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.service;
+
+public interface WriterService {
+    void writeToFile(String pathToFile, String report);
```

Comment:
```suggestion
    void writeToFile(String pathToFile, String data);
```

Because it can write any String, non only in the terms of `report`.